langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -79,7 +79,7 @@ public const int UNSIGNED8_MAX_VALUE = 255;
 // XXX this will panic for the most negative value (-2^63 is an int but +2^63 isn't)
 // consistent with policy on integer overflow
 
-# Returns absolute value of an int.
+# Returns absolute value of an `int` value.
```

Comment:
Same comment re: not emphasizing everywhere. I'm OK with either approach as long as it is consistent.